src/app/users/users.component.html

```diff
@@ -103,10 +103,7 @@ <h1 i18n>All Users</h1>
           <a
             *ngIf="element.doc.roles.length > 0 && !filterAssociated"
             (click)="setRoles(element.doc, element.doc.isUserAdmin ? element.doc.oldRoles : ['manager'])"
-            mat-raised-button color="primary" i18n>
-            <!--Convert boolean to number for i18n-->
-            {+element.doc.isUserAdmin, select, 1 {Demote} 0 {Promote}}
-          </a>
+            mat-raised-button color="primary" i18n>{+element.doc.isUserAdmin, select, 1 {Demote} 0 {Promote}}</a>

```

Comment:
We might want to remove this one as well from i18n. Also we need to update style guide README to not use i18n if we do not have any string inside element. Will create new issue for that.

Reply:
This is the `i18n` select syntax for having a few strings translated based on a variable value, so it is valid.